Header1

IvanChudiyovuch opened the pull request:
Додав кілька компонентівб глянь чи може бути така структура папок

IvanChudiyovuch committed:
Header1

Додав кілька компонентів

dev-kucha commented:
@IvanChudiyovuch, наче ж, домовлялися на styled components робити.
А у тебе css модулі.

IvanChudiyovuch committed:
Heder

Виправив стилі на styled component. Подивись чи структура компонентів підходить?

Tripoli80 committed:
Merge branch 'main' into header1
